extensions/amp-sticky-ad/1.0/amp-sticky-ad.js

```diff
@@ -57,6 +63,32 @@ class AmpStickyAd extends AMP.BaseElement {
   buildCallback() {
     this.viewport_ = this.getViewport();
     this.element.classList.add('i-amphtml-sticky-ad-layout');
+
+    // Setting padding-bottom to avoid iPhone home bar
+    if (isExperimentOn(this.win, 'sticky-ad-padding-bottom')) {
+      const experimentInfoMap = /** @type {!Object<string,
+        !../../../src/experiments.ExperimentInfo>} */ ({
+        [[STICKY_AD_PADDING_BOTTOM_EXP.id]]: {
+          isTrafficEligible: () => true,
+          branches: [
+            [STICKY_AD_PADDING_BOTTOM_EXP.control],
+            [STICKY_AD_PADDING_BOTTOM_EXP.experiment],
+          ],
+        },
+      });
+      randomlySelectUnsetExperiments(this.win, experimentInfoMap);
+      if (
+        getExperimentBranch(this.win, STICKY_AD_PADDING_BOTTOM_EXP.id) ==
+        STICKY_AD_PADDING_BOTTOM_EXP.experiment
+      ) {
+        setStyle(
+          this.element,
+          'padding-bottom',
```

Comment:
Should this be `bottom`? If so, you can override with `setImportantStyles` (ick) or use an `!important` in a css file.

Reply:
If we use `bottom`, the sticky ads will be floating above the bottom. We want to use padding to expand the ads container to contain the extra unusable space.